growth/too/flask.py

```diff
@@ -4,11 +4,22 @@
 from flask import Flask
 from flask_humanize import Humanize
 from werkzeug.routing import BaseConverter
+from astropy.io import ascii
 
 
 # Application object
 app = Flask(__name__, instance_relative_config=True)
+
+# read db access information
+db_access_path = os.path.join(app.instance_path, 'db_access.csv')
```

Comment:
Please take a look at what we previously did for authenticating with growth-db and emulate that. (The growthdb module was recently deleted, but it's still in the history.)

Reply:
Our flask.py module contained this:

```
app.config['SQLALCHEMY_BINDS'] = {
    'growthdb': 'postgresql:///growthtest'
}
```

and then there was a file in secrets/application.cf.d/databases.cfg that contained this:

```
SQLALCHEMY_BINDS = {
    'growthdb': 'postgresql://real-user:real-password@real-host/real-database'
}
```

Reply:
OK, thanks that makes sense, I will commit those credentials separately. (Could you grant me permissions to the relevant repo, when you have a chance?)

Reply:
@lpsinger, I've been having some difficulty getting the decam db connected the way you suggested. I tried setting app.config['SQLALCHEMY_BINDS'] = {'decam_db': 'postgresql:///growthtest'} in flask.py
and SQLALCHEMY_BINDS = {
   'decam_db': 'postgresql://real_user:real_password@real_host:real_port/real_db_name'
} in secrets/application.cf.d/databases.cfg
but it doesn't recognize 'growthtest'. How should I initialize decam_db in flask.py?